web/src/testdata.test.tsx

```diff
@@ -68,12 +68,14 @@ function oneResource(): any {
       Edits: ["main.go"],
       StartTime: ts,
     },
-    PodName: "vigoda-pod",
-    PodCreationTime: ts,
-    PodStatus: "Running",
-    PodRestarts: 1,
-    Endpoints: ["1.2.3.4:8080"],
-    PodLog: "1\n2\n3\n4\nabe vigoda is now dead\n5\n6\n7\n8\n",
+    ResourceInfo: {
```

Comment:
All of this testdata was wrong: pod specific information is always nested under `ResourceInfo`.